scripts/whos.pl

```diff
@@ -70,14 +77,32 @@ sub sig_whos
 
    @who = split(/\s+/,$msg,9);
 
-   if($who[4] =~ /$SERVER_NAME/)
-   {
-      Irssi::printformat(MSGLEVEL_CRAP, 'whos',$who[1], $who[5],$who[6], $who[7], $who[2], $who[3], $who[8]);
-   }
+   $res{$server->{tag}}->{result}->{$who[5]}=[@who];
 }
 
 sub sig_whosend
 {
    my ($server, $msg, $nick, $address, $target) = @_;
+   if ($res{$server->{tag}}->{regex} eq '') {
+      $res{$server->{tag}}->{regex}= $nick;
+   }
+   Irssi::print("%9regex:%n".$res{$server->{tag}}->{regex},MSGLEVEL_CRAP);
```

Comment:
if you want to remain this line, use "Irssi::printformat" and register a format for it